e2e-tests/version-service/run

```diff
@@ -26,24 +26,22 @@ function check_telemetry_transfer() {
 		| $sed -e "s#image:\$#image: $IMAGE_MONGOD#" \
 		| $sed -e "s#image:.*-pmm\$#image: $IMAGE_PMM#" \
 		| $sed -e "s#image:.*-backup\$#image: $IMAGE_BACKUP#" \
-		| $sed -e "s#image: .*-mongod[34].*#image: $IMAGE_MONGOD#" \
-		| $sed -e "s#image: .*-mongodb:[34].*#image: $IMAGE_MONGOD#" \
+		| $sed -e "s#image: .*-mongod[345].*#image: $IMAGE_MONGOD#" \
+		| $sed -e "s#image: .*-mongodb:[345].*#image: $IMAGE_MONGOD#" \
 		| kubectl_bin apply -f -
 
 	desc 'check if Pod is started'
 	wait_for_running "${cluster}-rs0" 1
 	sleep 20
-	compare_kubectl "statefulset/${cluster}-rs0"
 
```

Comment:
why did we delete it? if we don't use it at all could we delete files from compare folder?

Reply:
@nmarukovich I am not sure why it is needed because we just create the cluster and check  'telemetry' using custom CV. But up to you.

Reply:
we can skip it no problem